app/form_objects/support/gias_info_form.rb

```diff
@@ -0,0 +1,29 @@
+class Support::GiasInfoForm
```

Comment:
I'm not sure about this being a FormObject - it doesn't submit any data, doesn't have any attributes, and doesn't have any need for validations. Would it be better to call this something else (maybe a Presenter?) or even just call the wrapped methods directly on the SchoolUpdateService from the controller?

Reply:
Good shout. Originally though this might need to hold search params etc.